packages/mobile/src/services/offline-downloader/offline-storage.ts

```diff
@@ -219,18 +220,19 @@ export const verifyTrack = async (
   return booleanResults.every((result) => result)
 }
 
-/** Debugging method to clear all downloaded content */
-export const purgeAllDownloads = async () => {
+export const purgeAllDownloads = async (withLogs?: boolean) => {
   const trackIds = await listTracks()
-  console.log(`Before purge:`)
+  if (withLogs) {
+    console.log(`Before purge:`)
+  }
   await readDirRec(downloadsRoot)
   await RNFS.unlink(downloadsRoot)
   await RNFS.mkdir(downloadsRoot)
-  console.log(`After purge:`)
+  if (withLogs) {
+    console.log(`After purge:`)
+  }
   await readDirRec(downloadsRoot)
-  trackIds.forEach((trackId) => {
-    store.dispatch(unloadTrack(trackId))
-  })
+  store.dispatch(unloadTracks(trackIds))
```

Comment:
We should also get the collectionIds and call `offlineDownloadsActions.removeCollection` on each of them.

I wrote this before you could download collections.

Reply:
so given that, ill just remove this unloadTracks dispatch, cause the removeDownloads reducer sets the offline-download state to initialState

Reply:
yeah that works
unlink of the root should delete everything

Come to think of it, can you confirm if you haven't that this doesn't break the directory? (i.e. that downloads still work after this gets called) I think the mkdir I had on 230 should handle that, but want to make sure.